Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json

```diff
@@ -33,6 +41,10 @@
             "updated": 1609869554268,
             "translations": [
                 {
+                    "language": "TR",
+                    "text": "Özellikler",
+                    "updated": 1645082938811
+                  {
                     "language": "RU",
                     "text": "Свойства",
```

Comment:
There is the closing bracket with a comma after it missing.
},
{

You can see how on other files.

Reply:
I hope it's ok..I will not translate again.. Promise :=)))

Reply:
@prepheadrus I hope will keep translating. You have translated a lot and nearly all files look great and are adding value to superalgos. I think i didn`t exactly tell you where to add:
},
{
Thats my fault. Just so i can merge this pull request and by that you getting rid of troubles, can you replace the conflicting file by the one from the repository and app.contribute then? By this the "original" version gets pushed and the conflicting file gets replaced. Later on you can add the translation of this one file back.
We are very close to target :+1:

Reply:
I downloaded the original file from the develop branch and am sending it again. I hope it works this time.